.github/workflows/ci.yml

```
@@ -29,8 +29,7 @@ jobs:
       matrix:
         crate: ${{ fromJSON(needs.gather_published_crates.outputs.members) }}
     steps:
-      - name: Install Protoc
-        run: sudo apt-get install -y protobuf-compiler
+      - run: sudo apt-get install zlib1g-dev --yes
```

Also let's add a comment here for future us.
```suggestion
      - name: Install zlib for `libp2p-websocket` # https://github.com/paritytech/soketto/issues/72
        run: sudo apt-get install zlib1g-dev --yes
```